packages/babel/test/fixtures/transformation/optimisation.react.constant-elements/inline-elements/expected.js

```diff
@@ -23,4 +27,4 @@ function render() {
   return function () {
     return _ref2;
   };
-}
+}
```

Comment:
I uncommented the fs.writeFileSync in the test runner to make these files but I can undo these whitespace changes if you want.